doc-content/enterprise-only/openshift/apb-deploy-proc.adoc

```diff
@@ -29,7 +29,18 @@ IMPORTANT: If you want to deploy an environment with immutable servers and a mon
 *** If you deployed the *Immutable Server - Monitor* option and want to connect the server to the monitoring infrastructure:
 **** Under *Router integration*, the service name of the `rhpam-immutable-mon-smartrouter` service.
 **** Under *Controller integration*, the service name of the `rhpam-immutable-mon-rhpamcentrmon` service and the admin username and password that you set in the *Immutable Server - KIE Process Server* option.
-
++
+[IMPORTANT]
+====
+Avoid each of the following combinations of settings. These combinations produce an invalid environment.
+
+* *Process server*>**Database type** `H2` and *Process server*>**Number of replicas** exceeding `1`.
+* *Process server*>**Database type** `External` and *Process server*>**Sets of Process Servers** exceeding `1`.
+* *Red Hat - Single Sign-On* or *LDAP* configured and  *Process server*>**Sets of Process Servers** exceeding `1`.
```

Comment:
Even if the provisioner creates the client name/secret manually how will they pass these names to the KIE servers? Or do you mean, in fact, automatic creation? Also I believe @jakubschwan gave me these limitations so I really need his comments here. Removing the LDAP one though.

Reply:
If users create SSO clients manually, it is still not possible to configure it for more than one Process server sets in APB because there is env. variable only for one sso client [see here](https://github.com/ansibleplaybookbundle/rhpam-apb/blob/master/apb.yml/#L1073). This variable configures same sso client to all Process server sets. So after the deployment users must change configuration of each Process server set and replace the sso client.